platforms/s390x/README.md

```diff
@@ -0,0 +1,41 @@
+# Connect on s390x
+Most steps to run connect on s390x are the same as described in the main [README](../../README.md), with a few exceptions noted below.
```

Comment:
can we summarize why the manual steps are required and the 3.7 version of the docker compose yml is required? This could also help us track the things that need to occur so that we can remove the platform specific instructions.